lib/trivial.nix

```diff
@@ -69,13 +37,16 @@ rec {
   and = x: y: x && y;
 
   /* bitwise “and” */
-  bitAnd = builtins.bitAnd or zipIntBits (a: b: if a==1 && b==1 then 1 else 0);
+  bitAnd = builtins.bitAnd
+    or zipIntBits (a: b: if a==1 && b==1 then 1 else 0);
```

Comment:
may even inline ```or (import ./zip-int-bits.nix)```  here?
yes, import 3 times, but in the most case there will be no import at all because ```bitAnd``` is in ```builtins```

Reply:
Okay!

Reply:
Done.